src/sentry/static/sentry/app/stores/organizationStore.jsx

```diff
@@ -0,0 +1,30 @@
+import Reflux from 'reflux';
+
+import OrganizationActions from 'app/actions/organizationActions';
+import OrganizationsActions from 'app/actions/organizationsActions';
+
+const OrganizationStore = Reflux.createStore({
+  listenables: OrganizationActions,
+
+  init() {
+    this.reset();
+
+    // also listen to updates from OrganizationsActions triggered from settings
```

Comment:
Should settings call `OrganizationAction.update` instead?

Reply:
Mmm I could change it, is it peculiar that a store is listening to actions from a separate store? If I change it should I have the settings components call both actions or could I put `OrganizationAction.update` inside the `updateOrganization` function of `OrganizationActionsCreator` (or is that also a bit of a weird pattern)

Reply:
It's not peculiar to listen to other actions, actions are not necessarily 1:1 with a store. The reason I bring this up is more that I think it's probably wrong how we use it in `master`.

I would call `OrganizationAction.update` inside of `updateOrganization` so that we don't break the existing code (but I imagine in the future we'll fix the org settings as well so that eventually we will not use `organizationsAction.update`.

Reply:
Or actually, just only use `OrganizationsAction.update`

Reply:
gotcha, okay I'll change it to have `updateOrganization` call `OrganizationAction.update` and have the `OrganizationStore` store only listen to that one action.